app/off.py

```diff
@@ -19,9 +20,15 @@
 async def fetch_quality(source_url):
     """Function to fetch data-quality"""
     async with aiohttp.ClientSession(headers=header) as session:
-        quality_url = f"{source_url}/data-quality-errors.json"
-        async with session.get(quality_url) as resp:
-            return await resp.json()
+        try:
+            quality_url = f"{source_url}/data-quality-errors.json"
+            async with session.get(quality_url) as resp:
+                return await resp.json()
+
+        except ContentTypeError as e:
+            quality_url = f"{e.request_info.real_url}.json"
+            async with session.get(quality_url) as resp:
+                return await resp.json()
```

Comment:
It's a bit weird to do this on ContentTypeError (that is let the resp.json work)

Normally session.get should follow redirects (allow_redirects parameter is True by default). It would be interesting to know why it does not work.

Reply:
Oh sorry my comment is invalid, the problem is a bug where redirect removes the "json" extension. So your code makes sense.

Still my other comment holds (just use the right URL, this is a change that have been introduced in product opener, and thus the canonical url is now this one.)